web/src/features/fbaCalculator/components/map/FBAMap.tsx

```diff
@@ -58,6 +59,23 @@ const buildHFILayers = () => {
   })
 }
 
+const buildBCTileLayer = (extent: number[]) => {
+  return new Tile({
+    extent,
+    opacity: 0.5,
+    preload: Infinity,
+    source: new TileWMS({
+      url: 'https://openmaps.gov.bc.ca/geo/pub/wms',
+      params: {
+        LAYERS: 'WHSE_LEGAL_ADMIN_BOUNDARIES.DRP_MOF_FIRE_CENTRES_SP',
```

Comment:
Can we have some extra comments here about this layer? Basically a layer description? I never know what layers are just outlines, do they have a fill colour? Do they have text labels? I always have to discover this through experimentation, it would be real nice to at a glance understand exactly what this layer is.